distribution/docker/druid.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
```

Comment:
If I'm following this correctly, it is assuming the basic tarball for distribution. Can you add a comment at the top of this file that this shell script is meant to help run the stock tarball? and what the expected parameters are?